agent/js/package.json

```diff
@@ -2,9 +2,19 @@
   "name": "agent-protocol",
   "version": "0.1.0",
   "description": "API for interacting with Agent",
+  "homepage": "https://e2b.dev",
+  "license": "MIT",
+  "author": {
+    "name": "e2b",
+    "email": "hello@e2b.dev",
+    "url": "https://e2b.dev"
+  },
+  "bugs": "https://github.com/e2b-dev/agent-protocol/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/e2b-dev/agent-protocol"
+  },
   "main": "./dist/index.js",
-  "module": "./dist/index.m.js",
-  "unpkg": "dist/index.umd.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "prepublishOnly": "pnpm build",
```

Comment:
I think the pnpm here should be npm?

Reply:
I had just been using what was in the repo since the start. If you want it to be npm it can be, although I believe pnpm is slightly faster from some discussions I've had. Up to you!

Reply:
I saw the GH action (#11) and figured I'd go ahead and change it to npm. :+1: